src/Npgsql/Replication/PgOutput/PgOutputReplicationOptions.cs

```diff
@@ -7,7 +7,7 @@ namespace Npgsql.Replication.PgOutput
     /// <summary>
     /// Options to be passed to the pgoutput plugin
     /// </summary>
-    public class PgOutputReplicationOptions
+    public class PgOutputReplicationOptions : IEquatable<PgOutputReplicationOptions>
```

Comment:
We could make them C# 9.0 `record` types and get the `Equals()` and `GetHashCode()` methods for free.
I think that would only work for .NET 5.0 and above though.

Reply:
As discussed offline, they're great candidates but unfortunately we'd need to have two implementations anyway (because of pre-.NET 5.0), and the value of making these records don't seem that big. But it's a possibility indeed.